README.md

```diff
@@ -1,2 +1,30 @@
 # My Niteo Career
-My [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1) and [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2).
+
+Every Nitean creates a repository like this one in their own GitHub account. The reason we do it like this is that the repository might contain personal information and we want every Nitean to be the master of their personal information. If they leave Niteo, they revoke access to their career repo and noone can read it any more. It also gives Niteans clarity into who exactly can read the One on One's notes: only the people listed as collaborators to the repo.
+
+The "meat" of the repo are two issues:
+* [One on One notes](https://github.com/niteoweb/my-niteo-career/issues/1): Every time you have a One on One. you or your senior writes a report here. The issues provides a template for the report.
+* [Career Path](https://github.com/niteoweb/my-niteo-career/issues/2): Every time your have a One on One, you talk about the weeks since the last One on One and highlight potentials wins in this issue and recognize the steps required to get promoted to the next level. 
+
+## How to create the Niteo Career repo for yourself
+
+### Create the repo
+
+1. Go to https://import.github.com/.
+1. For `Your old repository’s clone URL` enter `https://github.com/niteoweb/my-niteo-career.git`.
+1. For `Name` enter `my-niteo-career`.
```

Comment:
Since we're writing instructions, isn't "now" the best time to suggest a different name? "Career" perhaps.